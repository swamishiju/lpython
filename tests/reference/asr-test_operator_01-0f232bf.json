{
    "basename": "asr-test_operator_01-0f232bf",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_operator_01.py",
    "infile_hash": "fe3fc1c23c2f63265b758b4d5d757943c1462811c5bc5e7a14abb64d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_operator_01-0f232bf.stderr",
    "stderr_hash": "753b12cfe4313636360ed650c3f92527fc6e89a0d1669489f2792aed",
    "returncode": 2
}