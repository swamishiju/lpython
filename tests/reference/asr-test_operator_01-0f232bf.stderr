semantic error: Unary operator '-' not supported for type derived type
 --> tests/errors/test_operator_01.py:9:11
  |
9 |     print(-a)
  |           ^^ 
